trillian.proto

```diff
@@ -251,6 +235,15 @@ message SignedLogRoot {
 
   // log_root_signature is the raw signature over log_root.
   bytes log_root_signature = 9;
+
+  reserved 1, 2, 3, 4, 5, 6, 7;
+  reserved "key_hint";
```

Comment:
Optionally:
Add comments with int tags? E.g.:
```
  reserved "log_id"; // 5
```
Or sort by tag instead of lexicographically?

Reply:
I don't think this really brings any useful information? The important thing is that these should no longer be used, and keeping them in lexicographical order is easiest to check and maintain.

Reply:
Fair enough.